### DISCUSSION TITLE
Source Change Date

### DISCUSSION THREAD

**Discussion by JDubbTX** [codefori/vscode-ibmi#223] [Q&A] [answered]
Is there a way to see source change dates?  Rdi implements this via context menu, but I believe there is also a global setting:

![image](https://user-images.githubusercontent.com/56802156/126678621-68531cc7-135d-4048-b922-65c4f5486c27.png)

**Comment by worksofliam** [comment]
It would be awesome, wouldn't it.

Monaco has a `lineNumbers` property which lets us over-write the line numbers with the source dates. We used that in ILEditor 2.

I can't seem to find that in the VSCode API, which is why I haven't done it.

**Reply by JDubbTX** [reply]
Other ideas - display it at the bottom, ala Git Blame.  

Searching by change date is also a common necessity in my line of work and forces me to go to the green screen if I don't have RDi.

**Reply by worksofliam** [reply]
@JDubbTX currently the source date support is limited to those users that support development of the extension (code.ileditor.dev).

Searching / filtering source by source date might absolutely be something I could add to the source date extension.

**Reply by worksofliam** [reply]
Also, maybe we could show the source date in a code lens (similar to show git lens works).

**Comment by worksofliam** [comment, marked as answer]
Hey @JDubbTX - this is implemented and release in 0.7.1.

**Reply by JDubbTX** [reply]
Thanks.  I will check it out soon.

**Comment by szsascha** [comment]
Short general question regarding this:

I'm currently on PUB400 and didn't see the line dates in SEU by default. Maybe because the default configuration is to use fully free RPG.

Does somebody know what I have to configure to use fixed length to show line dates?
